operator/main.go

```diff
@@ -65,6 +69,9 @@ func main() {
 	flag.DurationVar(&gracePeriod, "grace-period", 10*time.Second, "The duration the operator will wait before checking a release's status after reconciling")
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
+	flag.StringVar(&slackChannel, "slack-channel", "us-east-1", "The channel to send Slack notifications to")
```

Comment:
I don't think `us-east-1` is a good default value here, probably some leftover copypaste

Reply:
oops, yep